contents.css

```diff
@@ -115,7 +115,6 @@ figure
 	text-align: center;
 	border: solid 1px #ccc;
 	border-radius: 2px;
-	background: rgba(0,0,0,0.05);
```

Comment:
This change deletes background color from captioned images (`image` widgets).
<img width="322" alt="image widget without background" src="https://user-images.githubusercontent.com/1078728/35682020-9a90c4a6-075f-11e8-903c-da37c27cd8ee.png">